src/ui/layout/message-layout.svelte

```diff
@@ -81,6 +81,10 @@
     .message-layout {
       flex-grow: 1;
     }
+
+    .message-layout-footer {
```

Comment:
Will check if it causes any unexpected diffs on Chromatic

Reply:
As the error text description grew, I noticed the button got to close to the text and added this spacing.